app/routes/events/components/EventDetail/index.tsx

```diff
@@ -529,13 +524,13 @@ const EventDetail = () => {
                       toggleModal={toggleModal}
                       registrations={registrations}
                       currentRegistration={currentRegistration}
-                      skeleton={showSkeleton}
+                      skeleton={fetching && !registrations}
                     />
                     <AttendanceStatus
                       toggleModal={toggleModal}
                       pools={pools}
                       legacyRegistrationCount={event.legacyRegistrationCount}
-                      skeleton={showSkeleton}
+                      skeleton={fetching && !registrations}
```

Comment:
<img width="1137" alt="image" src="https://github.com/webkom/lego-webapp/assets/69514187/bd9c56ee-c6e7-4c8c-acad-3ebdccc843b7">

Here the event was retrieved from the store (because the event was already fetched from the overview page), while the registrations still need to be fetched